applications/swimming_DEM_application/python_scripts/swimming_DEM_algorithm.py

```diff
@@ -97,13 +97,27 @@ def SetFluidAlgorithm(self):
     def SetDispersePhaseAlgorithm(self):
         import dem_main_script_ready_for_coupling as DEM_algorithm
         self.disperse_phase_solution = DEM_algorithm.Solution(self.pp)
+        
+    def ReadDispersePhaseAndCouplingParameters(self):
+        
```

Comment:
Hi @GuillermoCasas . As we spoke, I put this here. However, I hate that this code is a repetition of the function that reads the json file for the dem and for swimming dem. I cannot call the actual code because the DEM solution isn't even created., that's why I copied it here. At least the Json file is read just once.